frontend-next-migration/src/entities/Hero/ui/BarChart/BarChart.module.scss

```diff
@@ -0,0 +1,130 @@
+.resistance{
```

Comment:
remove hardcoding for values. Values can be whatever , we should define that via props(see below samples)